2-0-samples.md

```diff
@@ -303,6 +303,10 @@ hidden from the MISO interface.
 #TODO Content here!
 
 
+# 2.6 Scanning Matrix Tubes into a Box
```

Comment:
Will become 2.5 since I removed receiving libraries and added information in "Troubleshooting" instead. Please clarify that Stocks should be added to the inbox (assuming that's the intention here)